packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -33,108 +51,97 @@ const Wrapper = styled.div`
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
     padding: ${GEL_MARGIN_ABOVE_400PX};
   }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding-top: ${GEL_SPACING_QUAD};
+    padding-bottom: ${GEL_SPACING_QUAD};
+  }
 `;
 
-/*
- * The '&::after' below is to ensure that the background colour covers the
- * banner as the inner elements are float. The alernative is to have
- * another div inside. This implementation mirrors the current orbit banner.
- */
 const CenterWrapper = styled.div`
-  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+  max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX};
   margin: 0 auto;
 
-  &::after {
-    content: '\\0020';
-    display: block;
-    height: 0;
-    clear: both;
-    overflow: hidden;
-    visibility: hidden;
+  a {
+    color: ${C_CONSENT_ACTION};
+    text-decoration: underline;
+    text-decoration-color: ${C_PEBBLE};
+
+    ${hoverFocusStyles}
   }
 `;
 
 const Title = styled.h2`
-  ${({ script }) => script && getGreatPrimer(script)};
+  ${({ script }) => script && getDoublePica(script)};
   color: ${C_WHITE};
   font-weight: 700;
   padding: 0;
   margin: 0;
-
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    float: ${ltrRtl('left', 'right')};
-    ${ltrRtl('margin-right: 3.5%;', 'margin-left: 3.5%;')}
-    width: 22%;
-  }
 `;
 
 /*
  * The '& li + li' below allows for styling every `li` element except the first.
  */
 const Options = styled.ul`
```

Comment:
Please can we add an attribute of role list to the ul please? This will resinstate the list semantics on VoiceOver, which it removes due to seeing list stlye none on the CSS